flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/sink/KafkaUtil.java

```diff
@@ -56,16 +56,16 @@ private KafkaUtil() {}
      */
     public static KafkaContainer createKafkaContainer(String dockerImageVersion, Logger logger) {
         String logLevel;
-        if (logger.isErrorEnabled()) {
-            logLevel = "ERROR";
-        } else if (logger.isTraceEnabled()) {
+        if (logger.isTraceEnabled()) {
```

Comment:
Good catch. Just to double-check: We need this change because the levels are not mutually exclusive and rather a higher log level implies that all log levels below are also enabled.

Reply:
Exactly, so previously it would always default to ERROR in case any log level was set